default/content/settings.json

```diff
@@ -44,7 +44,6 @@
         "preset": "Default",
         "add_bos_token": true,
         "stopping_strings": [],
-        "truncation_length": 2048,
```

Comment:
This removal is correct, ye?
Because int he preset manager below `truncation_length` was added, on the same level as `stopping_strings` was too.
Is that setting gone now, or isn't it? What this mean?

Reply:
Expected. `truncation_length` didn't do anything, but was in the default settings, and since presets are saved from the `textgen_settings` object, even dead properties would get there. So this both stops it for new and existing installs.